core/lib/src/models/study_schedule/study_schedule.dart

```diff
@@ -10,8 +10,11 @@ class StudySchedule {
   int phaseDuration = 7;
   bool includeBaseline = true;
   PhaseSequence sequence = PhaseSequence.alternating;
+  String sequenceCustom;
 
-  StudySchedule();
+  StudySchedule({
+    this.sequenceCustom = 'ABAB',
+  });
```

Comment:
Will this be BREAKING? Database can't be used anymore when sequenceCustom is set. Why?